codebuild/spec/buildspec_asan.yml

```diff
@@ -0,0 +1,63 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+# This buildspec runs on an Ubuntu22 image. That configuration is a property of
+# the codebuild job itself.
+
+# Codebuild's matrix jobs have non-differentiated names so use batch-list
+# instead.
+batch:
+  build-list:
```

Comment:
I don't see any fips libcryptos here-- do we currently run asan on any fips libcryptos? Which libcryptos does s2nGeneralBatch currently run against?

Reply:
We actually only run against 3 libcrypto's currently: openssl-1.1.1, openssl-3.0, and openssl-1.0.2. I added aws-lc because that's our happy path, but otherwise just tried to match omnibus buildspec.